src/main/java/org/jboss/snowdrop/licenses/LicensesGenerator.java

```diff
@@ -120,4 +118,15 @@ private Artifact gavToArtifact(Gav gav) {
                 gav.getType());
     }
 
+
+    protected static LicenseSummaryFactory createLicenseSummaryFactory(GeneratorProperties generatorProperties) {
+        LicenseSanitiser noopLicenseSanitiser = new NoopLicenseSanitiser();
+        LicenseSanitiser aliasLicenseSanitiser = new AliasLicenseSanitiser(LICENSE_NAMES_FILE, noopLicenseSanitiser);
+        LicenseSanitiser externalLicenseSanitiser =
```

Comment:
Because you have the generatorProperties here, you could check if service url is available and decide if this sanitiser should be created. This way you could keep implementation of the sanitiser simpler.

Reply:
Done